src/content/en/updates/2020/07/referrer-policy-new-chrome-default.md

```diff
@@ -0,0 +1,237 @@
+JavaScript using `document.referrer`.
+
+Up until recently,
+[`no-referrer-when-downgrade`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy)
+has been a widespread default policy across browsers. But now many browsers are in some stage of
+[moving to more privacy-enhancing
+defaults](https://web.dev/referrer-best-practices/#default-referrer-policies-in-browsers).
+
+**[Starting in Chrome 85](https://blog.chromium.org/2020/07/chrome-85-upload-streaming-human.html),
+Chrome is switching its default policy from `no-referrer-when-downgrade` to
+`strict-origin-when-cross-origin`.**
+
+This means that if no policy is set for your website, Chrome will use
+`strict-origin-when-cross-origin` by default. Note that you can still set a policy of your choice;
+this change will only have an effect on websites that have no policy set.
+
+<div class="clearfix"></div>
+
+Note: this step to help reduce silent cross-site user tracking is part of a larger initiative: the
+Privacy Sandbox. Check [Digging into the Privacy
+Sandbox](https://web.dev/digging-into-the-privacy-sandbox/) for more details.
+
+### What does this change mean?
+
+`strict-origin-when-cross-origin` offers more **privacy**. With this policy, only the
+[origin](https://web.dev/same-site-same-origin/#origin) is sent in the `Referer` header of
+cross-origin requests.
+
+This prevents leaks of private data that may be accessible from other parts of the full URL such as
+the path and query string.
+
+<figure>
+ <img src="/web/updates/images/2020/07/referrer-policy-default-02.png" alt="Diagram: Referer sent
+      depending on the policy, for a cross-origin request."/>
+ <figcaption>
+   <em>Referer sent (and document.referrer) for a cross-origin request, depending on the
+   policy.</em>
+ </figcaption>
+</figure>
+
+For example:
+
+Cross-origin request, sent from https://site-one.example/**stuff/detail?tag=red** to
+https://site-two.example/…:
+
+- With `no-referrer-when-downgrade`: Referer: https://site-one.example/**stuff/detail?tag=red**.
+- With `strict-origin-when-cross-origin`: Referer: https://site-one.example/.
+
+### What stays the same?
+
+- Like `no-referrer-when-downgrade`, `strict-origin-when-cross-origin` is **secure**: no referrer
+  (`Referer` header and `document.referrer`) is present when the request is made from an HTTPS
+  origin (secure) to an HTTP one (insecure). This way, if your website uses HTTPS ([if not, make it
+  a priority](https://web.dev/why-https-matters/)), your website's URLs won't leak in non-HTTPS
+  requests—because anyone on the network can see these, so this would expose your users to
+  man-in-the-middle-attacks.
+- Within the same origin, the `Referer` header value is the full URL.
+
+**Example:**
+
+Same-origin request, sent from https://site-one.example/**stuff/detail?tag=red** that has a
+`strict-origin-when-cross-origin` policy, to https://site-one.example/:<br/> Referer:
+https://site-one.example/**stuff/detail?tag=red**.
```

Comment:
```suggestion
For example:

Same-origin request, sent from https://site-one.example/**stuff/detail?tag=red** to 
https://site-one.example/…:

 - With `strict-origin-when-cross-origin`: Referer: https://site-one.example/**stuff/detail?tag=red**
```

Double check that I got this right. I tried to mirror the same style/format you used about for the cross origin request (lines 103-109)

Reply:
Looks better indeed, done.